packages/edit-site/src/components/template-details/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getTemplateInfo } from '../../utils';
+
+export default function TemplateDetails( { template } ) {
+	if ( ! template ) {
+		return null;
+	}
+	const { title, description } = getTemplateInfo( template );
+
+	return (
+		<div className="edit-site-template-details">
+			<p className="edit-site-template-details__heading">
+				{ __( 'Template details' ) }
+			</p>
+
+			{ title && (
+				<p>
+					{ sprintf(
+						/* translators: %s: Name of the template. */
+						_x( 'Name: %s' ),
```

Comment:
Not quite sure I understand the purpose of `_x` from the I18n library. I might be mistaken, but from what I see, it is meant to provide more context for strings that _could_ be translated similarly, but _aren't_ because of context ([src](https://developer.wordpress.org/reference/functions/_x/)). 

If that's the case, why do we include a contextual comment on line 26 instead of a second argument to `_x`?